dipy/denoise/noise_estimate.py

```diff
@@ -274,6 +296,23 @@ def estimate_sigma(arr, disable_background_masking=False):
     k[1, 1, 0] = 1
     k[1, 1, 2] = 1
 
+    # Precomputed factor from Koay 2006, this corrects the bias of magnitude image
+    correction_factor = {1: 0.42920367320510366,
+                         4: 0.4834941393603609,
+                         6: 0.4891759468548269,
+                         8: 0.49195420135894175,
+                        12: 0.4946862482541263,
+                        16: 0.4960339908122364,
+                        20: 0.4968365823718557,
+                        24: 0.49736907650825657,
+                        32: 0.49803177052530145,
+                        64: 0.49901964176235936}
+
+    if N in correction_factor:
+        factor = correction_factor[N]
+    else:
+        factor = correction_factor[1]
```

Comment:
If the user provides a non-sense value of `N`, I think that it would make more sense to raise an error, rather than assuming that they meant `1`.